experimenter/experimenter/theme/static_src/src/styles.css

```diff
@@ -1,3 +1,63 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+
+[x-cloak] { 
+    display: none !important;
+}
+
+a[role=tab] {
+    border-bottom: 1px solid transparent;
+    transition: all var(--transition);
+    min-width: 120px;
+    text-align: center;
+    user-select: none;
+    width: 100%;
+    padding: 4px;
+    color: #938f8f;
+}
+a[role=tab].active {
+    color: #242D4Eff;
+    border-radius: 0;
+}
+a[role=tab]:focus {
+    background: transparent;
+}
+div.grid div {
+    min-height: 200px;
+    animation-duration: 0.45s;
+    -webkit-animation-duration: 0.45s;
+    animation-fill-mode: both;
+    -webkit-animation-fill-mode: both;
+    -webkit-animation-name: slideIn;
```

Comment:
These do not need -webkit- prefix. See:

* https://developer.mozilla.org/en-US/docs/Web/CSS/animation-fill-mode
* https://developer.mozilla.org/en-US/docs/Web/CSS/animation-name
* https://developer.mozilla.org/en-US/docs/Web/CSS/animation-duration

They are supported since 2015